CHANGELOG.md

```diff
@@ -3058,12 +3132,64 @@ Here is an example of the renaming for the `<ChartsTooltip />` component.
 - [core] Update release instructions as per v7 configuration (#10962) @MBilalShafi
 - [license] Correctly throw errors (#10924) @oliviertassinari
 
+## 6.19.11
```

Comment:
You might need to resync v6 changelog after the v6 package release.

Reply:
Do you mean [this](https://github.com/mui/mui-x/pull/12832#discussion_r1570841044) or something else?

Reply:
Yes